install/install.go

```diff
@@ -756,35 +793,59 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	}
 
 	fmt.Print("❌   Cluster Role ...\n")
-	if err := c.K8sClientset.RbacV1().ClusterRoles().Delete(context.Background(), deployments.KubeArmorClusterRoleName, metav1.DeleteOptions{}); err != nil {
-		if !strings.Contains(err.Error(), "not found") {
-			return err
+	clusterRoleList, err := c.K8sClientset.RbacV1().ClusterRoles().List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app"})
```

Comment:
Is this backwards compatible?
We don't have the label selector in the current Cluster Role right?